CONTRIBUTING.md

```diff
@@ -195,48 +195,35 @@ installed (`cargo install hyperfine`).
 
 ## Configuring `rust-analyzer`
 
-To configure `rust-analyzer` and VS Code for working on Miri, save the following
-to `.vscode/settings.json` in your local Miri clone:
-
-```json
-{
-    "rust-analyzer.rustc.source": "discover",
-    "rust-analyzer.linkedProjects": [
-        "Cargo.toml",
-        "cargo-miri/Cargo.toml",
-        "miri-script/Cargo.toml",
-    ],
-    "rust-analyzer.check.invocationLocation": "root",
-    "rust-analyzer.check.invocationStrategy": "once",
-    "rust-analyzer.check.overrideCommand": [
-        "env",
-        "MIRI_AUTO_OPS=no",
-        "./miri",
-        "clippy", // make this `check` when working with a locally built rustc
-        "--message-format=json",
-    ],
-    // Contrary to what the name suggests, this also affects proc macros.
-    "rust-analyzer.cargo.buildScripts.invocationLocation": "root",
-    "rust-analyzer.cargo.buildScripts.invocationStrategy": "once",
-    "rust-analyzer.cargo.buildScripts.overrideCommand": [
-        "env",
-        "MIRI_AUTO_OPS=no",
-        "./miri",
-        "check",
-        "--message-format=json",
-    ],
-}
-```
+To configure `rust-analyzer` and the IDE for working on Miri, use one of the provided
+configuration files according to the instructions below.
+
+### Visual Studio Code
+
+Copy [`src/etc/rust_analyzer_settings.json`] to `.vscode/settings.json` in the project root directory.
+
+### Helix
+
+Copy [`src/etc/rust_analyzer_helix.toml`] to `.helix/languages.toml` in the project root directory.
+
+Since working on Miri requires a custom toolchain consider running `./miri toolchain` with the
+`-c rust-analyzer` flag, or set the `RUSTUP_TOOLCHAIN` environment variable according to
+[the documentation](https://rust-analyzer.github.io/manual.html#toolchain).
+
+> #### Hint
+>
+> To keep the `rust-analyzer` configuration up-to-date, make a symbolic link to the provided file.
 
 > #### Note
```

Comment:
This now looks like the note is specific for Helix, which is not the case.

Any suggestion for how to avoid that confusion?

Reply:
I haven't decided how it is better yet. Moving it before the IDE paragraphs makes it too bold and the follow up seems disconnected. I will think about it.

Reply:
Seems better now. Are you OK with it?

Reply:
I would suggest to move the hint (about using a symlink) to the first section, the one about VSCode. People reading to-down will hopefully notice it then even if they are using a different IDE.

The note about "If you are building Miri with a locally built rustc" could move into the "Advanced configuration" section.

Reply:
> People reading to-down will hopefully notice it then even if they are using a different IDE.

Maybe above the `### Visual Studio Code`? Then it wouldn't look like it applies only to VS Code, but also for Helix.

Reply:
Then it is confusing because there haven't been any "files" yet that this comment refers to.

Reply:
Make a symbolic link to the provided files below? Still the same place, but it point that the files are inside the IDE and Helix sections.

If you're not satisfied with that idea, I will just follow your suggestion to get it merged.

Reply:
I don't understand the suggestion. But a forward reference will just be confusing I think, this hint can't come before the first concrete IDE configuration section.